src/backend/src/services/change-requests.services.ts

```diff
@@ -132,7 +132,7 @@ export default class ChangeRequestsService {
     } else if (foundCR.type === CR_Type.ACTIVATION && foundCR.activationChangeRequest && accepted) {
       await this.reviewActivationChangeRequest(foundCR, reviewer);
     }
-
+    const includeArgs = getChangeRequestQueryArgs(organization.organizationId).include;
```

Comment:
This can just be in the `include: {}`, doesn't need a variable if it is only being used once